mwana/apps/export/views.py

```diff
@@ -0,0 +1,132 @@
+# vim: ai ts=4 sts=4 et sw=4
+# Create your views here.
+import psycopg2
+from mwana import localsettings as settings
+from django.http import HttpResponse
+import json
+from mwana.apps.locations.models import Location
+from collections import OrderedDict
+
+
+def get_turnaround(request):
+    '''
+    key_value = ""
+    if ((request.method == 'GET') and ('key_value' in request.GET)):
+        key_value = request.GET['key_value']
+    '''
+    connection = psycopg2.connect(host=settings.DATABASES['default']['HOST'], database=settings.DATABASES['default']['NAME'],\
```

Comment:
Hi Paul,

Could you clarify on Trevor's final 2 queries?

Trevor, is that a small enough change (the DB connector) that you can fix
as you see fit for the Mwana codebase?

On Mon, Feb 1, 2016 at 6:18 PM Trevor Sinkala <notifications@github.com>
wrote:

> In mwana/apps/export/views.py
> <https://github.com/mwana/mwana/pull/192#discussion_r51439053>:
>
> > +# Create your views here.
> > +import psycopg2
> > +from mwana import localsettings as settings
> > +from django.http import HttpResponse
> > +import json
> > +from mwana.apps.locations.models import Location
> > +from collections import OrderedDict
> > +
> > +
> > +def get_turnaround(request):
> > +    '''
> > +    key_value = ""
> > +    if ((request.method == 'GET') and ('key_value' in request.GET)):
> > +        key_value = request.GET['key_value']
> > +    '''
> > +    connection = psycopg2.connect(host=settings.DATABASES['default']['HOST'], database=settings.DATABASES['default']['NAME'],\
>
> And didn't get feedback on this one
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mwana/mwana/pull/192/files#r51439053>.
>